sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -137,6 +231,35 @@ SUBROUTINE ALLOCATE_ALL()
         allocate(cnvctdetmflx(im,jsta_2l:jend_2u,lm))
         allocate(cnvctzgdrag(im,jsta_2l:jend_2u,lm))
         allocate(cnvctmgdrag(im,jsta_2l:jend_2u,lm))
+!Initialization
+        do l=1,lm
+          do j=jsta_2l,jend_2u
+            do i=1,im
+              vdifftt(i,j,l)=spval
+              vdiffmois(i,j,l)=spval
+              dconvmois(i,j,l)=spval
+              sconvmois(i,j,l)=spval
+              nradtt(i,j,l)=spval
+              o3vdiff(i,j,l)=spval
+              o3prod(i,j,l)=spval
+              o3tndy(i,j,l)=spval
+              mwpv(i,j,l)=spval
+              unknown(i,j,l)=spval
+              vdiffzacce(i,j,l)=spval
+              zgdrag(i,j,l)=spval
+              cnvctummixing(i,j,l)=spval
+              vdiffmacce(i,j,l)=spval
+              mgdrag(i,j,l)=spval
+              cnvctvmmixing(i,j,l)=spval
+              ncnvctcfrac(i,j,l)=spval
+              cnvctumflx(i,j,l)=spval
+              cnvctdmflx(i,j,l)=spval
+              cnvctdetmflx(i,j,l)=spval
+              cnvctzgdrag(i,j,l)=spval
+              cnvctmgdrag(i,j,l)=spval
+            enddo
+          enddo
+        enddo 
```

Comment:
@WenMeng-NOAA @junwang-noaa I feel these legacy D3D variables may not be used anymore.  Perhaps we should remove them in the near future?

Reply:
@HuiyaChuang-NOAA Could you specify names of these D3D variables?